config/manager/kustomization.yaml

```diff
@@ -13,5 +13,5 @@ kind: Kustomization
 
 images:
 - name: controller
-  newName: ghcr.io/nutanix-cloud-native/cluster-api-provider-nutanix/controller
-  newTag: latest
+  newName: ko.local/cluster-api-provider-nutanix
+  newTag: e2e-432658dd426691c79d167a7502036904f19d87d2
```

Comment:
this is pointing GitHub container registry, when we create releases it uses these as manifests so let's not change this. Instead, set env var LOCAL_IMAGE_REGISTRY to your internal OCI registry before running tests instead.

Reply:
Thanks for helping catch this out. I checked in this by accident.

Fixed.